src/collections/blog/2022-02-25-layer5-collab-with-openforce/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Layer5 collaboration with OpenForce 2022"
+date: 2022-02-25 
+author: Shebeul Inyang
+thumbnail: 
+category: "Community"
+tags:
+ - Community
+ - Events
+ - OpenSource
+
+featured: true
+published: true
+---
+
+

```

Comment:
@unnati914 Please use BlogWrapper here also import image hero image for the card